lingcorpora/corpora/emk_corpus.py

```diff
@@ -85,10 +85,10 @@ def parse_result(self,result):
         else:
             lc = result.select('span.nott')[0].string.strip()
             rc = result.select('span.nott')[-1].string.strip()
-        final_kws = result.select('td.kw div.token span.nott')[0].string.strip()
+        final_kws = result.select('div.token span.nott')[0].string.strip()
```

Comment:
`td.kw` is no longer present in the html. It was not found and therefore there appeared `IndexError`